### PR TITLE
Add lib to set all workers as unschedulable

### PR DESCRIPTION
**Pull request by kasturinarra**
Adding a library to set all the worker nodes in the cluster to unschedulable state.

### CODE REVIEW

**Comment by kasturinarra** [comment]
@zhouying7780 can you please help review ? thanks

File: features/step_definitions/node.rb [hunk 1]
```diff
@@ -639,3 +639,13 @@
     raise "cannot restore node '#{cb[cb_name]}'" unless @result[:success]
   }
 end
+
+Given /^I set all worker nodes status to unschedulable$/ do
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  nodes = env.nodes.select { |n| n.is_worker? }
+  nodes.each do |node, schedulable|
+  opts = { :node_name => node.name}
```
Comment (pruan-rht) [inline features/step_definitions/node.rb]: indentation

File: features/step_definitions/node.rb [hunk 2]
```diff
@@ -639,3 +639,13 @@
     raise "cannot restore node '#{cb[cb_name]}'" unless @result[:success]
   }
 end
+
+Given /^I set all worker nodes status to unschedulable$/ do
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  nodes = env.nodes.select { |n| n.is_worker? }
+  nodes.each do |node, schedulable|
```
Comment (pruan-rht) [inline features/step_definitions/node.rb]: the variable `schedulable` is declared but never used.

**Comment by kasturinarra** [comment]
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/758/console

**Pull request merged**